src/services/user/types.ts

```
@@ -23,3 +23,23 @@ export type UserOwnedData = {
+  description: string;
+  email: string;
+  location: string;
+  header: string;
+  contenthash: string;
+  links: {
+    website: {
+      link: string;
+      handle: string;
+    };
+  };
+  social: {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)